cloudstack/data_source_cloudstack_network_offering.go

```diff
@@ -54,6 +54,64 @@ func dataSourceCloudstackNetworkOffering() *schema.Resource {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
+			"network_rate": {
+				Type:     schema.TypeInt,
+				Computed: true,
```

Comment:
Need to add `ForceNew:    true,` for all these options to ensure new network offering is created. As of now, if I create an offering and then change the `network_rate`, I see the plugin crashing.